mod_reforged/hooks/skills/racial/spider_racial.nut

```diff
@@ -0,0 +1,32 @@
+::mods_hookExactClass("skills/racial/spider_racial", function(o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+		this.m.Name = "Spider";
+		this.m.Description = "";	// Vanilla has "TODO" written here. We don't want that to display
+		this.m.Icon = "ui/orientation/spider_01_orientation.png";
+		this.m.IsHidden = false;
+		this.addSkillType(::Const.SkillType.StatusEffect);	// We now want this effect to show up on the enemies
+	}
+
+	o.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.extend([
+			{
+				id = 10,
+				type = "text",
+                icon = "ui/icons/bravery.png",
+				text = "Resove is increased by " + ::MSU.Text.colorGreen("+3") + " for every other ally on the battlefield of the same faction"
+			},
+			{
+				id = 11,
+				type = "text",
+                icon = "ui/icons/direct_damage.png",
+				text = "When attacking a webbed target, increase the damage that ignores armor by " + ::MSU.Text.colorGreen("100%")
```

Comment:
When attacking a webbed target, the damage that ignores armor is increased by 100%